gitlab_runner/datadog_checks/gitlab_runner/data/conf.yaml.example

```diff
@@ -9,7 +9,6 @@ init_config:
     - ci_docker_machines_provider_machine_states
     - ci_runner_builds
     - ci_runner_errors
-    - ci_runner_version_info
```

Comment:
If this is still in the metrics list, the version won't be parsed.
When removed, the version is parsed and the metric is sent, that's why I put the Added label.
However since the user may have to change its configuration file, maybe we should do it differently

Reply:
Not sure I understood your comment here. What do you mean by "it will not be parsed"?

If this is because our OpenMetricsCheck doesn't allow the `metadata_metric_name` to be one of the `metrics`, then we can probably programmatically force `ci_runner_version_info` out of `metrics` if defined.

This way no config file change is required and we can proceed with dropping it from here because it will be submitted in any case.

Alternatively we may want to only submit version metadata if `ci_runner_version_info` is whitelisted but I don't think it's desirable. IIRC `allowed_metrics` was a way to not submit too many metrics, it's fine if we force-submit `ci_runner_version_info` I think.

Reply:
> If this is because our OpenMetricsCheck doesn't allow the metadata_metric_name to be one of the metrics, then we can probably programmatically force ci_runner_version_info out of metrics if defined.

Yes exactly. I think that would be a good solution, thanks 👍